src/NBitcoin.Tests/ChainTests.cs

```diff
@@ -554,7 +555,7 @@ public ChainedHeader AppendBlock(ChainedHeader previous, params ConcurrentChain[
             uint nonce = RandomUtils.GetUInt32();
             foreach (ConcurrentChain chain in chains)
             {
-                Block block = TestUtils.CreateFakeBlock(this.network.CreateTransaction());
+                Block block = TestUtils.CreateFakeBlock(this.network, this.network.CreateTransaction());
```

Comment:
you can just call `TestUtils.CreateFakeBlock(this.network);` here.

Reply:
Fixed :)